pkg/cloud/tags/tags.go

```diff
@@ -115,7 +115,11 @@ func WithEC2(ec2client ec2iface.EC2API) BuilderOption {
 			}
 
 			_, err := ec2client.CreateTags(createTagsInput)
-			return errors.Wrapf(err, "failed to tag resource %q in cluster %q", params.ResourceID, params.ClusterName)
+			if err != nil {
+				return errors.Wrapf(err, "failed to tag resource %q in cluster %q", params.ResourceID, params.ClusterName)
+			}
+			errmsg := fmt.Sprintf("failed to tag resource %q in cluster %q", params.ResourceID, params.ClusterName)
+			return errors.Errorf(errmsg)
```

Comment:
nit: Same comment as above

Reply:
I reverted this back to the way it is in upstream since no error will return nil by errors.Wrapf and it was causing tests to fail